backend/open_webui/routers/auths.py

```
@@ -4,7 +4,7 @@
 import datetime
 import logging
 from open_webui.internal.db import get_db
-from open_webui.models.users import User, UserModel
+from open_webui.models.users import User, UserModel, Users
```

Ruff (0.8.2)

`open_webui.models.users.UserModel` imported but unused

Remove unused import: `open_webui.models.users.UserModel`

(F401)